modules/javascript/pages/ionic.adoc

```diff
@@ -1,77 +1,34 @@
 = Ionic
 :page-partials:
-:page-edition: 
+:page-edition: Public Preview
 :page-toclevels: 2@
-:description: Using Couchbase Lite with Javascript Ionic applications
+:description: The Ionic Capacitor plugin for Couchbase Lite allows developers to leverage Couchbase Lite within cross-platform applications in JavaScript for iOS and Android. 
 
 include::partial$_set_page_context_for_javascript.adoc[]
 
-:param-abstract: If you are building mobile or desktop applications using web technologies such as JavaScript/HTML/CSS, there are options for using Couchbase Lite as an embedded database within your app.
-:param-related: xref:cordova.adoc[] | xref:react.adoc[]
-include::{root-partials}_show_page_header_block.adoc[]
+== Overview
 
-.Support Changes
-[IMPORTANT]
+NOTE: Developers who are currently using the https://ionic.io/docs/couchbase-lite[previous version of the Ionic plugin] should migrate to the newer version of the plugin documented https://cbl-ionic.dev[here].
```

Comment:
Not a big issue, but I think if a link is directing users to an external site, it would be polite to give them some clue as to where they're going. Something like

```suggestion
NOTE: Developers who are currently using the https://ionic.io/docs/couchbase-lite[previous version of the Ionic plugin] should migrate to the newer version of the plugin documented at https://cbl-ionic.dev[Couchbase Lite for Ionic Capacitor].
```